test/spanner.ts

```diff
@@ -4671,7 +4671,7 @@ describe('Spanner with mock server', () => {
             MultiplexedSessionRetry: 'precommitToken',
             precommitToken: {
               precommitToken: fakeRetryToken,
-              seqNum: 1,
+              seqNum: 100,
```

Comment:
I dont understand why are there changes in this test, why should code changes impact this?